assembl/static2/js/app/components/debate/brightMirror/sideComment/sideCommentBox.jsx

```diff
@@ -224,18 +263,31 @@ class DumbSideCommentBox extends React.Component<Props, State> {
     );
   };
 
+  toggleReplying = () => {
+    this.setState(state => ({ replying: !state.replying }));
+  };
+
+  renderActionFooter = () => {
+    const { userCanReply } = this.props;
+    const { extractIndex } = this.state;
+    const currentExtract = this.getCurrentExtract(extractIndex);
+    const currentComment = this.getCurrentComment(currentExtract);
+    const hasReply = !!this.getCurrentCommentReply(currentExtract, currentComment);
```

Comment:
Curious: why ```!!``` ?

Reply:
It's JS etiquette for turning a non-bool into a bool.

Reply:
oooh right